src/configmanager.cpp

```diff
@@ -111,6 +111,7 @@ bool ConfigManager::load()
 
 		integer[SQL_PORT] = getGlobalNumber(L, "mysqlPort", 3306);
 		integer[GAME_PORT] = getGlobalNumber(L, "gameProtocolPort", 7172);
+		integer[CAST_PORT] = getGlobalNumber(L, "loveCastProtocolPort", 7173);
```

Comment:
Files changes -> go to a line, put the marker next to a line number and click the blue + box :)